src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor_Minimal.cs

```diff
@@ -159,6 +159,20 @@ private void MinimallyQualify(INamedTypeSymbol symbol)
             AddNameAndTypeArgumentsOrParameters(symbol);
         }
 
+        protected bool InObjectInitializatioContext(INamedTypeSymbol symbol)
```

Comment:
@CyrusNajmabadi, by simplifying `NameBoundSuccessfullyToSameSymbol`, the test `SemanticQuickInfoSourceTests.ColorColorRangeVariable` fails since the QuickInfo for the range variable is `yield yield` rather than `N1.yield yield`. The former looks correct to me and is consistent with the QuickInfo for the range variable in other parts of the query. Do we need the namespace qualifier in this case?